command_model:
  type: velocity_and_yaw_const
  vel: 1.0
  yaw_rate: 0.1
  std_dev: [0.1, 0.6981317007977318]  # 40.0deg

motion_model:
  type: circle2d
  dt: 0.1
  std_dev: [0.1, 0.1, 0.6981317007977318, 0.1]

observation_model:
  type: gps_xy
  std_dev: [0.5, 0.5]
# type: gps_xyY
# std_dev: [0.2, 0.2, 0.1]
# type: rfid_xyd
# std_dev: [0.1, 0.1]
# max_range: 20.0
# id_list: [[10.0, 0.0],
#           [10.0, 10.0],
#           [0.0, 15.0],
#           [-5.0, 20.0]]

# パーティクルフィルタには未対応
sampler:
  type: uniform_dist
  prob: 0.50
# type: time_based_interval
# interval_sec: 1.0

bayesian_filter:
  type: ekf
# type: ukf
# alpha: 0.001
# beta: 2.0
# kappa: 0.0
# type: pf
# p_num: 100
# p_resample: 50
# resample_threshold: 50

plotter:
  type: simple
# type: with_landmark
